frontend/.eslintrc.js

```diff
@@ -9,6 +9,12 @@ module.exports = {
     // https://nextjs.org/docs/basic-features/eslint
     "next/core-web-vitals",
   ],
+  rules: {
```

Comment:
This is in next.js template now, added it here to prevent another ignore in this PR